### DISCUSSION TITLE
Two HA-Instances with two Tunnels and two different subdomains, but with only one domain

### DISCUSSION THREAD

**Discussion by pballhausen** [homeassistant-apps/app-cloudflared#459] [Ideas]
Is it possible for me to reach two different HA instances over a tunnel with just one Cloudflare account and a single domain?

I would like to access both my Home Assistant and my parents' Home Assistant via a subdomain. If I'm understanding correctly, the tunnels are always created under the name "Homeassistant" and therefore it doesn't work. Is it possible to also specify the name of the tunnel or to use the hostname of the Home Assistant?

**Comment by elcajon** [comment]
It is possible to create different tunnels with different IDs/names by setting the tunnel_name (add-on) configuration option to something different (homeassistant is only the default). 
Another option would be to create (and configure, see our docs for further information) the tunnel via the Cloudflare dashboard (just name it ha-parents e.g.) and pass the provided token to config option tunnel_token.
